src/styles/header.scss

```diff
@@ -0,0 +1,48 @@
+.menu {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  background-color: #ffffff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 30px;
+  justify-content: space-between;
+}
+
+.menu-items {
+  display: flex;
+  list-style: none;
+}
+
+.list {
+  text-decoration: none;
+  color: $navigation-color;
+  margin: 0 20px;
+  text-transform: uppercase;
+  transition: all 0.3s;
+
+  &:hover{
+    color: $active-navigation;
+  }
+}
+
+.list--number-5 {
```

Comment:
The class `list--number-5` does not adhere to the BEM naming convention. In BEM, modifiers should be appended to the block or element with a double hyphen, and the base class should be a valid BEM block or element. Consider renaming this class to follow the BEM convention.